Tasks/PublishBuildArtifacts/publishbuildartifacts.ts

```diff
@@ -96,6 +97,11 @@ async function run() {
                 // copy the files
                 let script: string = path.join(__dirname, 'Invoke-Robocopy.ps1');
                 let command: string = `& ${pathToScriptPSString(script)} -Source ${pathToRobocopyPSString(pathtoPublish)} -Target ${pathToRobocopyPSString(artifactPath)} -ParallelCount ${parallelCount}`
+                if (fs.statSync(pathtoPublish).isFile()) {
+                    let parentFolder = path.dirname(pathtoPublish);
```

Comment:
I think these `let`s can be `const`.

Reply:
Cool.  Next time if I add new strings I will keep this in mind.  For now I think using "let" looks more consistent with the rest of the file.